hstrat/genome_instrumentation/_HereditaryStratum.py

```diff
@@ -42,9 +43,13 @@ def __init__(
         """
         if deposition_rank is not None:
             self._deposition_rank = deposition_rank
-        self._differentia = random.randrange(2**differentia_bit_width)
-        if annotation is not None:
-            self._annotation = annotation
+
+        if differentia:
+            self._differentia = differentia
+        else:
+            self._differentia = random.randrange(2**differentia_bit_width)
+
+        self._annotation = annotation
```

Comment:
Why are we assigning `_annotation` if `annotation` is `None`? I think this is fine in principle, but why do we need to make the change now? (previously, we did not assign `_annotation = None`).

Reply:
Because if `annotation` is `None`, then `self._annotation` would not get defined at all, thus making `GetAnnotation()` error out. This way, it gets initialized to `None` and `GetAnnotation()` doesn't fail when called.